client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -36,7 +36,6 @@ function ReaderSubscriptionListItem( {
 	const siteIcon = get( site, 'icon.img' );
 	const feedIcon = get( feed, 'image' );
 	const streamUrl = getStreamUrl( feedId, siteId );
-	const siteUrl = url || getSiteUrl( { feed, site } );
```

Comment:
looks like for feed search, URL is the site url, but `subscribe_URL` is the actual feed URL. could we use that instead?